src/System.Management.Automation/engine/DscResourceInfo.cs

```diff
@@ -1,6 +1,6 @@
 //-----------------------------------------------------------------------
 // <copyright company="Microsoft">
-//    Copyright (C) 2013 Microsoft Corporation
+//    Copyright (c) 2013 Microsoft Corporation
```

Comment:
There are a bunch of entries that include the date. It should be removed and made consistent with the others shouldn't it? This comment applies to a bunch of other changes if it needs to be addressed.

Copyright (c) Microsoft Corporation, All rights reserved.

Reply:
Yeah, my understanding is that the date doesn't matter.  Will update.